src/screens/Customers/AddMeterReading.tsx

```diff
@@ -1,61 +1,101 @@
+  }
+
+  const startingMeterAmount = currentCustomer.startingMeterReading;
+  const startingMeterLastRecorded = currentCustomer.startingMeterLastChanged;
+
+
+  const handleSetMeterReadingAmount = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setMeterReadingAmount(event.target.value as string);
+  }
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    // Prevent page refresh on submit
+    event.preventDefault();
+
+    const currentReadingAmount = parseFloat(meterReadingAmount);
+    if (isNaN(currentReadingAmount)) {
+      setErrorMessage('Meter Reading Amount must be a number');
+      return;
+    } else {
+      setLoading(true);
+    }
+
+    // WARNING: Client-side Amount Billed is treated as a source of truth in order to 
+    // accomodate for offline functionality when a user adjusts a Customer's startingMeterAmount when offline
+    const meterReading = JSON.parse(JSON.stringify(EMPTY_METER_READING));
+    meterReading.reading = currentReadingAmount;
+    meterReading.amountBilled = calculateAmountBilled(currentReadingAmount - startingMeterAmount, tariffPlan);
+    meterReading.date = moment().toISOString();
+    meterReading.customerId = currentCustomer.id;
+    meterReading.billedById = userId;
+    meterReading.meterNumber = currentCustomer.meterNumber;
+
+    createMeterReadingAndUpdateCustomerBalance(meterReading, currentCustomer).then(history.goBack);
+  }
+
+  const cardInfo = [{
+    number: startingMeterAmount.toString(),
+    label: 'Current Reading',
+    unit: 'Kwh',
+    secondaryLabel: formatDateStringToLocal(startingMeterLastRecorded)
+  }]
 
   return (
     <BaseScreen title="Add Meter Reading" leftIcon="backNav">
-      <div className={classes.content}>
-        <form noValidate>
-          <div className={classes.secondaryText}>
-            <Typography variant="body1">Date Recorded</Typography>
-            <Typography variant="body1">00.00.0000</Typography>
-            <Typography variant="body1" style={{ marginTop: 10 }}>
-              Current Reading
-            </Typography>
-            <Typography variant="h2" style={{ fontSize: 22 }}>
-              0 kWh
-            </Typography>
-          </div>
-          <div className={classes.outlined}>
-            <Typography variant="body1" style={{ marginBottom: 0 }}>
-              Today
-            </Typography>
-            <Typography variant="body1" style={{ marginBottom: 15 }}>
-              00.00.0000
-            </Typography>
-            <TextField label={'New Meter Reading (kWh)'} id={'new-meter-reading'}/>
-          </div>
-          <Button label={'ADD'} />
-        </form>
+      <div className={classes.amountOwedContainer}>
+        <OutlinedCardList info={cardInfo} />
       </div>
+      <TextField label='New Meter Reading (kWh)' unit='kWh' id={'amount-metered'} placeholder='e.g. 100' type='number' onChange={handleSetMeterReadingAmount} />
+      <Button label={'ADD'} onClick={handleSubmit} loading={loading} />
```

Comment:
Nit: change to 'Add' since it's converted to uppercase by default. This makes it easier in case we ever want to change back button text to regular case app-wide.